artisatomic/readqubdata.py

```diff
@@ -60,7 +80,11 @@ def read_adf04(filepath, flog):
 
         upsilonheader = fleveltrans.readline().split()
         list_tempheaders = [f"upsT={x:}" for x in upsilonheader[2:]]
-        list_headers = ["upper", "lower", "ignore", *list_tempheaders]
+        # WII has upper and lower column order switched:
+        if atomic_number == 74 and ion_stage == 2:
+            list_headers = ["lower", "upper", "ignore", *list_tempheaders]
+        else:
+            list_headers = ["upper", "lower", "ignore", *list_tempheaders]
```

Comment:
a more robust way to do this (in case it occurs for other ions) is:
`lower, upper = min(lower, upper), max(lower, upper)`

Reply:
What I suggest only works when stepping through the actual values. I didn't realise you were just altering the columns order. Keep this the same, and later, we should ensure that we are checking this (lower < upper) just before writing out transition data.